sanctum-lst-list.toml

```diff
@@ -135,6 +135,30 @@ pool = "7thbAQrn9oRJsbz2CchoPSujGYpu4hCHnVrniBHupQsx"
 validator_list = "21RD9B26CJFTBt7FYJqXWycjbrALQXLFNxnT88y5pu8H"
 vote_account = "he1iusunGwqrNtafDtLdhsUQDFvo13z9sUa36PauBtk"
 
+[[sanctum_lst_list]]
+name = "FP SOL"
+symbol = "fpSOL"
```

Comment:
This is pineSOL? Looks like we should add a new CI check that makes sure token metadata matches name and symbol